src/NUnitFramework/framework/Constraints/Constraint.cs

```diff
@@ -156,20 +154,19 @@ protected virtual string GetStringRepresentation()
             foreach (object arg in Arguments)
             {
                 sb.Append(" ");
-                sb.Append(_displayable(arg));
+                sb.Append(Displayable(arg));
             }
 
             sb.Append(">");
 
             return sb.ToString();
-        }
-
-        private static string _displayable(object o)
```

Comment:
Similar as above. Moved to a static local function and renamed to fix a few compiler warnings. There's pattern matching updates within it too